token/cli/src/main.rs

```diff
@@ -2831,10 +2846,10 @@ fn process_command(
             let allow_unfunded_recipient = arg_matches.is_present("allow_empty_recipient")
                 || arg_matches.is_present("allow_unfunded_recipient");
 
-            let recipient_is_ata_owner = arg_matches.is_present("recipient_is_ata_owner");
+            let no_recipient_is_ata_owner = arg_matches.is_present("no_recipient_is_ata_owner");
             let use_unchecked_instruction = arg_matches.is_present("use_unchecked_instruction");
             let memo = value_t!(arg_matches, "memo", String).ok();
-
+            let recipient_is_ata_owner = arg_matches.is_present("recipient_is_ata_owner");
```

Comment:
Let's remove the `recipient_is_ata_owner` variable here, and also from `command_transfer` and instead have:
```
let no_recipient_is_ata_owner = arg_matches.is_present("no_recipient_is_ata_owner") || !arg_matches.is_present("recipient_is_ata_owner");
if arg_matches.is_present("recipient_is_ata_owner") {
    println_display(config, format!("recipient-is-ata-owner is now the default behavior. The option has been deprecated and will be removed in a future release."));
}
```